pkgs/development/tools/corundum/default.nix

```diff
@@ -0,0 +1,15 @@
+{ lib, rubyTool }:
+
+rubyTool {
+  name = "corundum-0.6.2";
```

Comment:
I think that's a really good idea. Doing it properly is going to take a dab more work - hold off merging this...